Source/DafnyLanguageServer/Language/DafnyLangParser.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
```

Comment:
Is there a test for the changes in this file? Having a test would make it easier for me to understand what the changes do.

Reply:
I'm working on a test, it's a good idea although there was no straightforward mechanism to test this.

Reply:
Test submitted in https://github.com/dafny-lang/dafny/pull/1694